clang/lib/StaticAnalyzer/Core/RangeConstraintManager.cpp

```diff
@@ -1939,11 +1956,8 @@ class RangeConstraintManager : public RangedConstraintManager {
   RangeSet::Factory F;
 
   RangeSet getRange(ProgramStateRef State, SymbolRef Sym);
-  RangeSet getRange(ProgramStateRef State, EquivalenceClass Class);
   ProgramStateRef setRange(ProgramStateRef State, SymbolRef Sym,
                            RangeSet Range);
-  ProgramStateRef setRange(ProgramStateRef State, EquivalenceClass Class,
-                           RangeSet Range);
```

Comment:
It [existed](https://github.com/llvm/llvm-project/commit/e63b488f2755f91e8147fd678ed525cf6ba007cc#diff-fe12a50c583524be9e6962cf5f85b16f246765c5ae108dba775b03a897c9f507R2017) a few years ago.
But probably it wasn't needed because an EQClass is implicit constructable from a SymbolRef by design. They are interchangeable. A Symbol always forms its own EQClass.

Reply:
This line was introduced by e63b488f2755f91e8147fd678ed525cf6ba007cc (`[analyzer][solver] Track symbol disequalities`), and removed this overload in 8474bb13c3270d4195a663013b95e6065075ce56 (`[analyzer][solver][NFC] Simplify function signatures`).

Reply:
I was confused a bit where was removed. The definition was actually removed in this commit:
d0d37fcc4e2965c4ccd3401f8bdf22935815136a (`[Analyzer][solver] Remove unused functions`)